libpod/runtime_img.go

```diff
@@ -270,26 +283,76 @@ func (k *Image) Decompose() error {
 	return nil
 }
 
-// HasImageLocal returns a bool true if the image is already pulled
-func (k *Image) HasImageLocal() bool {
+func getTags(nameInput string) (reference.NamedTagged, bool, error) {
+	inputRef, err := reference.Parse(nameInput)
+	if err != nil {
+		return nil, false, errors.Wrapf(err, "unable to obtain tag from input name")
+	}
+	tagged, isTagged := inputRef.(reference.NamedTagged)
+
+	return tagged, isTagged, nil
+
+}
+
+// GetLocalImageName returns  the name of the image if it is local.
+// it will return an empty string if not found.
```

Comment:
Can we return an error instead in the Not Found case? Libpod has ErrNoSuchImage, and checking for that is just as easy as checking for the empty string

Reply:
The comment doesnt match the code.  I updated it.